python/sglang/bench_serving.py

```diff
@@ -1578,7 +1582,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     parser.add_argument(
         "--warmup-requests",
         type=int,
-        default=20,
```

Comment:
Why change this?

Reply:
This parameter was added by me. Setting it to 1 ensures consistency with the initial warmup of 1 request.